lighthouse-core/gather/gather-runner.js

```diff
@@ -491,6 +505,13 @@ class GatherRunner {
 
     // Copy redirected URL to artifact.
     baseArtifacts.URL.finalUrl = passContext.url;
+    /* eslint-disable max-len */
+    if (!URL.equalWithExcludedFragments(baseArtifacts.URL.requestedUrl, baseArtifacts.URL.finalUrl)) {
```

Comment:
#10218 is going to throw a major wrench into all the places where we detect redirects like this 😆 

but a problem for another day